tests/test_job_management.py

```diff
@@ -1953,6 +1953,28 @@ def x(arg):
     assert len(my_globals['scwl_dict']['scw_list']) > max_length
 
 
+@pytest.mark.parametrize('sb_value', [25, 25., 25.64547871216879451687311211245117852145229614585985498212321])
+def test_spectral_parameter(dispatcher_live_fixture, sb_value):
```

Comment:
it's vital to have a complete test like this!

but I'd also add a unit test simply constructing parameters and getting default values from them. Just to capture how we handle conversions.

Reply:
test included!